net-mgmt/telegraf/src/opnsense/mvc/app/models/OPNsense/Telegraf/Output.xml

```diff
@@ -143,12 +143,12 @@
             <default>0</default>
             <Required>N</Required>
         </mqtt_enable>
-        <mqtt_topic_prefix type="TextField">
+        <mqtt_topic type="TextField">
             <default></default>
             <Required>N</Required>
-            <mask>/^([0-9a-zA-Z._\-]){1,128}$/u</mask>
-            <ValidationMessage>Only characters, numbers, a dot, underscore and hyphen allowed. Do not use more than 128 characters.</ValidationMessage>
-        </mqtt_topic_prefix>
+            <mask>/^([0-9a-zA-Z._\/\-]){1,200}$/u</mask>
```

Comment:
To use the following syntax {{ .TopicPrefix }}/{{ .Hostname }}/{{ .PluginName }}/{{ .Tag "tag_key" }} you must change the regex to <mask>/^([0-9a-zA-Z._\-\/{} ]){1,200}$/u</mask>

![image](https://github.com/opnsense/plugins/assets/30810603/ee78f75a-79e6-4863-b771-13e7d6e2c496)